contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
 </ComparisonTable>
 
-## Strengths of PostHog
+### Pricing
+
+**PostHog** charges for actual usage – i.e. product analytics events, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier** limits:
 
-![PostHog screenshot](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
+|                       | **Free usage per month** 
+|-----------------------|--------------------------
+| **Product analytics** | 1 million events        
+| **Session replay**    | 5,000 recordings         
+| **Feature flags**     | 1 million API requests
+| **A/B testing**       | 1 million API requests  
+| **Surveys**           | 250 responses            
 
-### Integrated session replay
+**Amplitude** charges based on monthly tracked users (MTUs). The cost of add owns, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.
 
-While Amplitude and PostHog share many features, PostHog has [session replays](/product/session-replay) built in as standard.
+Our free tiers are for life, which means you'll always get your first 1 million events (or 5,000 replays and so on) for free every month, no matter how many you end up using.
 
-Session replay is an incredibly powerful tool for understanding what people are actually doing in your product. The tight integration within PostHog means you can go from viewing a funnel insight to watching real users interact with the funnel, making it easy to diagnose problems and find solutions.
+Amplitude offers a free Starter plan, but it lacks many important features:
 
-Amplitude doesn't have session replay, so you have to run a third-party tool [like Hotjar or FullStory](/blog/best-open-source-session-replay-tools) as well – an added expense that lacks the tight integration afforded by a built-in app.
+<ComparisonTable column1="PostHog" column2="Amplitude">
+   <ComparisonRow column1="1 million events" column2="50k tracked users" feature="Free usage" description="Free tier usage each month" />
+   <ComparisonRow column1="Users" column2="Unlimited" feature="Unlimited" description="Active accounts" />
+   <ComparisonRow column1="Unlimited" column2="10 only" feature="Saved insights" description="Number of you charts you can create" />   
+   <ComparisonRow column1={true} column2={false} feature="Custom events" description="Create custom tracking events" />
+   <ComparisonRow column1={true} column2={false} feature="Formulas" description="Create custom insights using formulas" />
+   <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Release and control features" />
+   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
```

Comment:
```suggestion
   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
   <ComparisonRow column1={true} column2={false} feature="Billing limits" description="Set limits to avoid surprise bills" />
```